CHANGELOG.md

```diff
@@ -42,9 +42,9 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - Introduced a new crate `exonum-proto`. Trait `ProtobufConvert` is moved
   to this crate. (#1496)
 
-#### exonum-proto-derive
+#### protobuf-convert
```

Comment:
A name of the crate should be `exonum-protobuf-convert`, shouldn't it ?

Reply:
We discussed with @alekseysidorov, maybe it's possible to create general purpose crate not linked to exonum. To replace https://crates.io/crates/protobuf-convert

Reply:
If so then it should be out of `exonum` repo.